modules/ai/connect/ai.php

```diff
@@ -276,8 +276,8 @@ public function get_text_to_image( $data, $context, $request_ids ) {
 			'image/text-to-image',
 			[
 				self::PROMPT => $data['payload']['prompt'],
-				self::IMAGE_TYPE => $data['payload']['promptSettings'][ self::IMAGE_TYPE ] . '/' . $data['payload']['promptSettings'][ self::STYLE_PRESET ],
-				self::ASPECT_RATIO => $data['payload']['promptSettings'][ self::ASPECT_RATIO ],
+				self::IMAGE_TYPE => $data['payload']['settings'][ self::IMAGE_TYPE ] . '/' . $data['payload']['settings'][ self::STYLE_PRESET ],
```

Comment:
can you explain what was the issue?

Reply:
sure, the payload we're passing follows this structure -
```
{
   "payload":{
      "prompt":"...",
      "settings":{
         "image_type":"....",
         "style_preset":"....",
         "ratio":"....",
      }"..."
```
and the client expects 'promptSettings' key causing us to fail to extract the value